src/formio.wizard.js

```diff
@@ -22,11 +22,21 @@ export class FormioWizard extends FormioForm {
     // Validate the form builed, before go to the next page
     if (this.checkValidity()) {
       this.setPage(this.page + 1);
+      this.onNextPage({page: this.page, submission: this.submission});
     }
   }
 
   prevPage() {
     this.setPage(this.page - 1);
+    this.onPrevPage({page: this.page, submission: this.submission});
+  }
+
+  onPrevPage(submission) {
+    this.emit('prevPage', submission);
+  }
+
+  onNextPage(submission) {
```

Comment:
Why not just emit directly from the ```prevPage``` and ```nextPage``` methods without introducing two new superfluous methods.